metadata_mapper/validator/validator.py

```diff
@@ -781,7 +853,79 @@ def _ensure_is_list(value: Any) -> list[Any]:
         "field": "transcription",
         "validations": [Validator.content_match],
         "level": ValidationLogLevel.WARNING
-    }
+    },
+    {
+        "field": "campus_name",
+        "validations": [Validator.content_match],
+        "level": ValidationLogLevel.WARNING
+    },
+    {
+        "field": "campus_data",
+        "validations": [Validator.content_match],
+        "level": ValidationLogLevel.WARNING
+    },
+    {
+        "field": "reference_image_md5",
+        "validations": [Validator.content_match],
+        "level": ValidationLogLevel.WARNING
+    },
+    {
+        "field": "reference_image_dimensions",
+        "validations": [Validator.content_match],
+        "level": ValidationLogLevel.WARNING
+    },
+    {
+        "field": "created",
+        "validations": [Validator.content_match],
+        "level": ValidationLogLevel.WARNING
+    },
+    {
+        "field": "last_modified",
+        "validations": [Validator.content_match],
+        "level": ValidationLogLevel.WARNING
+    },
+    {
+        "field": "sort_date_start",
+        "validations": [Validator.content_match],
+        "level": ValidationLogLevel.WARNING
+    },
+    {
+        "field": "sort_date_end",
+        "validations": [Validator.content_match],
+        "level": ValidationLogLevel.WARNING
+    },
+    {
+        "field": "item_count",
+        "validations": [Validator.content_match],
+       "level": ValidationLogLevel.WARNING
```

Comment:
We need to do analysis on whether we have data in certain fields in Solr at some point, and decide how to clean that up if they're not being used anywhere. I thought we had decided to a straight port of the schema for now and do this analysis later? If not then we should discuss and prioritize that work, because it's not insignificant.

Reply:
Correcting mappers to get all these fields is not insignificant, either. Especially if we don't know where these field values come from. I think in some cases it is less work to determine that the field is meaningless, than it is to determine where the field's value comes from. 🤷‍♀️

Reply:
Ah right, I see your point. OK, I'll take a crack doing this analysis and will ping you with a new request for review when I've made changes.

Reply:
I guess one tactic could be to run the validator with these additions and see if there are errors. 🤷‍♀️